conf/runner.yaml

```diff
@@ -691,3 +691,8 @@ smtp:
     #auth:
     #    user: someuser
     #    pass: somepass
+
+slack:
+    username: tls-observatory
+    icon_emoji: ':telescope:'
```

Comment:
It isn't the emoji status in slack, it's simply the icon of slackbot when it sends the message. For example, if it was a monkey emoji, it would look like this (snipped from slack's documentation):
![image](https://cloud.githubusercontent.com/assets/6126493/25679168/288fbe6a-3001-11e7-99f2-2313e0483f76.png)